lib/CArtHandler.cpp

```diff
@@ -1249,6 +1272,11 @@ bool CArtifactSet::hasArt(
 	return getArtPosCount(aid, onlyWorn, searchBackpackAssemblies, allowLocked) > 0;
 }
 
+bool CArtifactSet::hasArtBackpack(ui32 aid) const

```

Comment:
Optional, outside of scope of this pull request:
replace `ui32 aid` with 'ArtifactID aid' where applicable,

Reply:
will do in another PR